user-guide/Cloud_Platform/Catalog/Change_email_cloud_account.md

```diff
@@ -19,6 +19,6 @@ To use a different email address for your existing dataminer.services account, y
 
 1. Visit [community.dataminer.services](https://community.dataminer.services/) with the new account.
 
-1. Contact ? to have your existing data transferred to the new account.
+1. Contact <back.office@skyline.be> to have your existing data transferred to the new account.
```

Comment:
Is it possible to use a hyperlink, so that the subject and body of the mail are already filled in straight away? (see hyperlink below)

[back.office@skyline.be](mailto:back.office@skyline.be?subject=Request%20to%20transfer%20my%20DataMiner%20Dojo%20data&body=Hi%2C%0D%0A%0D%0AI%20would%20like%20to%20transfer%20my%20DataMiner%20Dojo%20data.%0D%0AThe%20accounts%20involved%20are%20listed%20below%3A%0D%0A%0D%0AFrom%3A%0D%0A%3Cfill%20in%20email%20address%3E%0D%0A%0D%0ATo%20(new%20account)%3A%0D%0A%3Cfill%20in%20email%20address%3E%0D%0A)

Reply:
done!